plugins/restful/RestfulDataProviderDbQuery.php

```diff
@@ -379,51 +391,38 @@ public function update($id, $full_replace = FALSE) {
       'cl' => implode(',', $this->getIdColumn()),
       'id' => $id,
     ));
-    return $this->view($id, TRUE);
+
+    // @todo: do we need to re-form $id in case some of the values have changed?
```

Comment:
This was a left over comment from before I understood how the id processing worked; will be deleted.